shared/Views/Shared/Components/CourseDetails/Default.cshtml

```diff
@@ -74,7 +74,7 @@
       <div class="course-contents govuk-!-margin-bottom-8">
         <h2 class="govuk-heading-m">Contents</h2>
           <ul class="govuk-list govuk-list--dash">
-            @if (Model.PreviewMode || Model.HasSection(CourseDetailsSections.AboutTheCourse))
+            @if (Model.ShowAboutCourseSection)
```

Comment:
To keep the use of `PreviewMode` consistent I'd take the PreviewMode check out of `ShowAboutCourseSection` make this line `Model.PreviewMode ||  Model.ShowAboutCourseSection` and maybe rename ShowAbout... to something like `HasContentForAboutCourseSection`

suggestion